### PR TITLE
Rifex 261/clear search bar

### PR DESCRIPTION
**Pull request by mauricioirace**
Clear search bar when searching for a domain
- Search bar is cleared after searching for a domain that doesn't exist.
-  Search bar is not shown in domain detail page.

Note: A minor refactor was done in genericSearch, in order to handle the search text as a state property, and clear it when needed.

### CODE REVIEW

File: ui/app/rif/components/genericSearch/index.js [hunk 1]
```diff
@@ -20,19 +20,49 @@ class GenericSearch extends Component {
...
     const lowerString = v => String(v).toLowerCase()
     const lowerElement = lowerString(element);
     const lowerCriteria = lowerString(criteria);
     return lowerElement.includes(lowerCriteria);
   }
 
-  handleInput = async (e) => {
+  clear = () => {
+    this.setState({value: ''})
+    console.error('limpio');
```
Comment (rorolopetegui) [inline ui/app/rif/components/genericSearch/index.js]: Please delete this log, or we can have an error here?

**Review by rorolopetegui — CHANGES_REQUESTED** [review]
LGTM Just delete the error log that i think it will be always showed

**Review by JONAF2103 — CHANGES_REQUESTED** [review]
In general it looks good but please make those constant functions into the standar functions that we are using, instead of having somefunction = () => {} we are using the class member approach like this somefunction () {} inside the component class. With that you don't have to use bind in the constructor.

**Comment by mauricioirace** [comment]
> In general it looks good but please make those constant functions into the standar functions that we are using, instead of having somefunction = () => {} we are using the class member approach like this somefunction () {} inside the component class. With that you don't have to use bind in the constructor.

Ok, I'll change it. Anyway, I'm almost sure binding will still be needed

**Review by rorolopetegui — APPROVED** [review]
lgtm

**Review by JONAF2103 — APPROVED** [review]
LGTM

**Pull request merged**